corehq/apps/hqadmin/urls.py

```diff
@@ -53,6 +53,8 @@
     url(r'^auth_as/(?P<username>[^/]*)/(?P<domain>{})/$'.format(new_domain_re),
         AuthenticateAs.as_view(), name=AuthenticateAs.urlname),
     url(r'^superuser_management/$', SuperuserManagement.as_view(), name=SuperuserManagement.urlname),
+    url(r'^tombstone_management/$', TombstoneManagement.as_view(), name=TombstoneManagement.urlname),
```

Comment:
These should be linked in the admin menu.

Reply:
Ha, I don't blame you, but this is kind of security by obscurity. I think there's rarely if ever a case for magic links. We should permission things appropriately and/or have specific help text on the page about the risks. Since this is internal, I think adding candid help text is the way to go. In general users aren't good about reading help text, but the support team *is* good about it.